### ISSUE TITLE
force:apex:log:tail removed?

### ISSUE THREAD

**Issue by mkdjns** [forcedotcom/cli#1370]
I recently updated to 7.133.0 and, using one of my standard workflows, attempted to pipe `sfdx force:apex:log:tail` to a file. Instead of a log file full of debugs as I expected, I received the following-

```
 ›   Warning: force:apex:log:tail is not a sfdx command.
Did you mean force:apex:log:get? [y/n]: 
```

I see nothing in the release notes to indicate that `force:apex:log:tail` was removed, and https://developer.salesforce.com/docs/atlas.en-us.sfdx_cli_reference.meta/sfdx_cli_reference/cli_reference_force_apex.htm still references that command.

Was this command removed? Replaced with something else?

**Comment by mshanemc** [comment]
That like you have something wrong in your installation.

Can you run `sfdx version --verbose` and include the output like the issue template asks for?

**Comment by mkdjns** [comment]
Sorry, I didn't see issue templates when I posted my initial question.

Output of `sfdx version --verbose --json` below-

```
{
        "cliVersion": "sfdx-cli/7.133.0",
        "architecture": "darwin-x64",
        "nodeVersion": "node-v16.13.1",
        "pluginVersions": [
                "@oclif/plugin-autocomplete 0.3.0 (core)",
                "@oclif/plugin-commands 1.3.0 (core)",
                "@oclif/plugin-help 3.3.1 (core)",
                "@oclif/plugin-not-found 1.2.6 (core)",
                "@oclif/plugin-plugins 1.10.11 (core)",
                "@oclif/plugin-update 1.5.0 (core)",
                "@oclif/plugin-warn-if-update-available 1.7.3 (core)",
                "@oclif/plugin-which 1.0.4 (core)",
                "alias 1.2.0 (core)",
                "auth 1.8.0 (core)",
                "config 1.3.14 (core)",
                "data 0.6.8 (core)",
                "generator 1.2.1 (core)",
                "info 1.2.0 (core)",
                "org 1.11.0 (core)",
                "salesforcedx 50.12.0 (50.12.0)",
                "├─ salesforce-alm 53.7.2",
                "├─ schema 1.0.4",
                "├─ limits 1.0.4",
                "├─ @salesforce/sfdx-plugin-lwc-test 0.1.7",
                "├─ templates 50.1.0",
                "├─ apex 0.1.4",
                "└─ custom-metadata 1.0.11",
                "sfdx-cli 7.133.0 (core)",
                "source 1.7.0 (core)",
                "telemetry 1.4.0 (core)",
                "trust 1.1.0 (core)",
                "user 1.7.0 (core)"
        ],
        "osVersion": "Darwin 20.6.0"
}
```

I ran another update just after getting the above. The issue still exists, and now the output of the above command is-

```
{
        "cliVersion": "sfdx-cli/7.134.0",
        "architecture": "darwin-x64",
        "nodeVersion": "node-v16.13.2",
        "pluginVersions": [
                "@oclif/plugin-autocomplete 0.3.0 (core)",
                "@oclif/plugin-commands 1.3.0 (core)",
                "@oclif/plugin-help 3.3.1 (core)",
                "@oclif/plugin-not-found 1.2.6 (core)",
                "@oclif/plugin-plugins 1.10.11 (core)",
                "@oclif/plugin-update 1.5.0 (core)",
                "@oclif/plugin-warn-if-update-available 1.7.3 (core)",
                "@oclif/plugin-which 1.0.4 (core)",
                "alias 1.2.0 (core)",
                "auth 1.8.0 (core)",
                "config 1.3.14 (core)",
                "data 0.6.8 (core)",
                "generator 1.2.1 (core)",
                "info 1.2.0 (core)",
                "org 1.11.0 (core)",
                "salesforcedx 50.12.0 (50.12.0)",
                "├─ salesforce-alm 53.7.2",
                "├─ schema 1.0.4",
                "├─ limits 1.0.4",
                "├─ templates 50.1.0",
                "├─ @salesforce/sfdx-plugin-lwc-test 0.1.7",
                "├─ custom-metadata 1.0.11",
                "└─ apex 0.1.4",
                "sfdx-cli 7.134.0 (core)",
                "source 1.8.0 (core)",
                "telemetry 1.4.0 (core)",
                "trust 1.1.0 (core)",
                "user 1.7.0 (core)"
        ],
        "osVersion": "Darwin 20.6.0"
}
```

**Comment by mkdjns** [comment]
I think I have this sorted out. I noticed that the output of that command from a different Mac excluded the following output-

```
"salesforcedx 50.12.0 (50.12.0)",
"├─ salesforce-alm 53.7.2",
"├─ schema 1.0.4",
"├─ limits 1.0.4",
"├─ templates 50.1.0",
"├─ @salesforce/sfdx-plugin-lwc-test 0.1.7",
"├─ apex 0.1.4",
"└─ custom-metadata 1.0.11",
```
I ran `sfdx plugins` on the problem machine, and saw that node in the output. From there, I issued `sfdx plugins:uninstall salesforcedx`, and the issue is now resolved.
